src/Compilers/Core/Portable/Syntax/SyntaxToken.cs

```diff
@@ -212,7 +212,7 @@ public override string ToString()
         /// <remarks>The length of the returned string is always the same as FullSpan.Length</remarks>
         public string ToFullString()
         {
-            return _token != null ? _token.ToFullString() : string.Empty;
+            return _token?.ToFullString() ?? string.Empty;
         }
```

Comment:
one more. 
SyntaxToken struct is mostly just a wrapper for a green node which adds information about position and parent. The _token != null is just an unfortunate result of structs having default state.